src/App.css

```diff
@@ -14,8 +14,8 @@
 }
 
 .App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+  background-color: white;
```

Comment:
i'm stealing adrian's comment and saying to keep colors to either names, hex codes, or rgb lol

Reply:
I only do everything.